Description: Remove dependencies for the windows target
Author: Mike Hommey <glandium@debian.org>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: git-cinnabar/Cargo.toml
===================================================================
--- git-cinnabar.orig/Cargo.toml
+++ git-cinnabar/Cargo.toml
@@ -215,8 +215,6 @@ self-update = [
     "shared_child",
     "dep:tar",
     "dep:xz2",
-    "dep:zip",
-    "windows-sys/Win32_System_Threading",
 ]
 version-check = ["shared_child"]
 
@@ -234,25 +232,3 @@ optional = true
 [target."cfg(not(windows))".dependencies.xz2]
 version = "0.1"
 optional = true
-
-[target."cfg(windows)".dependencies.curl-sys]
-version = "0.4"
-features = [
-    "ssl",
-    "static-curl",
-]
-default-features = false
-
-[target."cfg(windows)".dependencies.libz-sys]
-version = "1"
-features = ["static"]
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.52"
-features = ["Win32_Foundation"]
-
-[target."cfg(windows)".dependencies.zip]
-version = "2"
-features = ["deflate-zlib"]
-optional = true
-default-features = false
